.env.local

```diff
@@ -0,0 +1,12 @@
+# this file is used by docker-compose-local.yml to set environment variables for the local development environment
+ACCESS_TOKEN_EXPIRE_SECONDS=60
+AWS_ACCESS_KEY_ID=test
+AWS_REGION_NAME=us-east-1
+AWS_SECRET_ACCESS_KEY=test
+DB_NAME=va-enp-api-db
+API_DB_READ_URI=postgresql+psycopg://postgres:LocalPassword@localhost:5432/notification_api
+API_DB_WRITE_URI=postgresql+psycopg://postgres:LocalPassword@localhost:5432/notification_api
+DB_READ_URI=postgresql+psycopg://postgres:LocalPassword@localhost:5433/va-enp-api-db
+DB_WRITE_URI=postgresql+psycopg://postgres:LocalPassword@localhost:5433/va-enp-api-db
```

Comment:
in the  db.init file, there is 
```
_engine_enp_read: AsyncEngine
_engine_enp_write: AsyncEngine
_engine_napi_read: AsyncEngine
_engine_napi_write: AsyncEngine
```

Considering updating these env vars should follow that format for clarity:
```
NAPI_DB_READ_URI=
NAPI_DB_WRITE_URI=
ENP_DB_READ_URI=
ENP_DB_WRITE_URI=
```